packages/node-cli/src/commands/actionresult/ActionResultBuilder.js

```diff
@@ -0,0 +1,35 @@
+/*
+** Copyright (c) 2019 Oracle and/or its affiliates.  All rights reserved.
+** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+*/
+'use strict';
+const ActionResult = require('../actionresult/ActionResult');
+const ActionResultStatus = require('../actionresult/ActionResultStatus');
+
+module.exports = class ActionResultBuilder {
```

Comment:
I don't think it makes sense that the builder needs a new instance in order to be used (`new ActionResultBuilder()`).
There are other builder patterns that seem more intuitive to use. See https://medium.com/@axelhadfeg/builder-pattern-using-javascript-and-es6-ec1539182e24

Reply:
ActionResultBuilder has been moved to ActionResult